cli/cmd/test_connection.go

```diff
@@ -22,6 +30,53 @@ cloudquery test-connection ./directory ./aws.yml ./pg.yml
 `
 )
 
+func getSyncTestConnectionAPIClient() (*cloudquery_api.ClientWithResponses, error) {
+	authClient := apiAuth.NewTokenClient()
+	if authClient.GetTokenType() != apiAuth.SyncTestConnectionAPIKey {
+		return nil, nil
+	}
+
+	token, err := authClient.GetToken()
+	if err != nil {
+		return nil, err
+	}
+	return api.NewClient(token.Value)
+}
+
+func updateSyncTestConnectionStatus(ctx context.Context, logger zerolog.Logger, status cloudquery_api.SyncTestConnectionStatus) {
+	apiClient, err := getSyncTestConnectionAPIClient()
+	if apiClient == nil {
```

Comment:
Erm, this won't handle errors

Reply:
🤦  https://github.com/cloudquery/cloudquery/pull/17212/commits/870f7da4dc75f71cece2eaaff0c8ab753f11444e